market/urls.py

```
@@ -1,5 +1,7 @@
 from django.urls import path
-from .views import SignUpView, IndexView, ProductListView, ProductUploadView, ProductDetailView, ContactSellerView, ProfileView, AboutView, ContactView, TeamView, ProductDeleteView, SellerProfileView, HomeView, MarketplaceView, add_to_cart, remove_from_cart, CartView, ReviewView, DeleteReviewView, CheckoutView, OrderConfirmationView, OrderHistoryView, OrderDetailView, NotificationsView, MarkNotificationReadView, WishlistView, add_to_wishlist, remove_from_wishlist, move_to_cart, LogoutView, UserFeedbackView, ArtistApplicationView, NewsletterSubscriptionView, ShippingUpdateView
+from django.contrib.auth.views import PasswordResetConfirmView, PasswordResetCompleteView
+from django.views.generic import TemplateView
+from .views import SignUpView, IndexView, ProductListView, ProductUploadView, ProductDetailView, ContactSellerView, ProfileView, AboutView, ContactView, TeamView, ProductDeleteView, SellerProfileView, HomeView, MarketplaceView, add_to_cart, remove_from_cart, CartView, ReviewView, DeleteReviewView, CheckoutView, OrderConfirmationView, OrderHistoryView, OrderDetailView, NotificationsView, MarkNotificationReadView, WishlistView, add_to_wishlist, remove_from_wishlist, move_to_cart, LogoutView, UserFeedbackView, ArtistApplicationView, NewsletterSubscriptionView, ShippingUpdateView, CustomPasswordResetView, ChangePasswordView, CustomLoginView
```

Ruff (0.12.2)

`.views.IndexView` imported but unused

Remove unused import

(F401)